promql/functions.go

```diff
@@ -598,10 +599,11 @@ func funcPredictLinear(vals []parser.Value, args parser.Expressions, enh *EvalNo
 func funcHistogramQuantile(vals []parser.Value, args parser.Expressions, enh *EvalNodeHelper) Vector {
 	q := vals[0].(Vector)[0].V
 	inVec := vals[1].(Vector)
-	sigf := enh.signatureFunc(false, excludedLabels...)
+	b := bytes.Buffer{}
```

Comment:
Why not use the one in enh?

Reply:
thanks, I missed that enh was being passed in